### PR TITLE
Dhfprod 817 default writer permissions

### PR DESCRIPTION
**Pull request by grechaw**
This is a work-in-progress and not ready for merging. 
@bsrikan for you to look at and peruse.

### CODE REVIEW

File: marklogic-data-hub/src/main/java/com/marklogic/hub/impl/FlowManagerImpl.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 public class FlowManagerImpl extends ResourceManager implements FlowManager {
     private static final String HUB_NS = "http://marklogic.com/data-hub";
     private static final String NAME = "ml:flow";
-
+t
     private DatabaseClient stagingClient;
```
Comment (bsrikan) [inline marklogic-data-hub/src/main/java/com/marklogic/hub/impl/FlowManagerImpl.java]: There is a compile error here.

Comment (grechaw) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/impl/FlowManagerImpl.java]: oh yes of course.  id found that in my editor, but i guess that was after it pushed.  ill fix momentarily

Comment (grechaw) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/impl/FlowManagerImpl.java]: resolved

File: marklogic-data-hub/gradle.properties [hunk 2]
```diff
@@ -52,6 +52,12 @@ mlUseRoxyTokenPrefix=false
 ## These values are used by the testing infrastructure.
 mlHubUserRole=data-hub-role
 mlHubUserName=data-hub-user
+mlHubAdminRole=hub-admin-rule
+mlHubAdminUser=hub-admin-user
```
Comment (bsrikan) [inline marklogic-data-hub/gradle.properties]: is this a typo. Should it be hub-admin-role?

Comment (grechaw) [reply marklogic-data-hub/gradle.properties]: yes most definitely a typo

**Comment by grechaw** [comment]
Remade this PR, and creating a code review request/updating issues

**Comment by grechaw** [comment]
@ehennum accepted this code in crucible

**Pull request merged**